components/src/core/Drawer/DrawerNavItem.tsx

```diff
@@ -17,7 +17,7 @@ import { NavItemSharedStyleProps, NavItemSharedStylePropTypes, SharedStyleProps,
 import clsx from 'clsx';
```

Comment:
Check line 12 of this file
import { InfoListItem, InfoListItemProps as PXBInfoListItemProps } from '../InfoListItem';